cmd/aro/operator.go

```diff
@@ -124,5 +125,12 @@ func operator(ctx context.Context, log *logrus.Entry) error {
 	// +kubebuilder:scaffold:builder
 
 	log.Info("starting manager")
+
+	s := liveness.New("8080") //creates a route on :8080 for healthz/ready for liveness probe
+
+	go func() {
+		log.Warn(s.ListenAndServe())
+	}()
```

Comment:
Not a fan of having code like this in `cmd` but for now not sure where to better put this. Somehow it might even be calling for its own dummy controller. But I don't think the amount of work is worth it :/ 
@troy0820 as much as this is simper, did you tested it?

Reply:
I have the same reservations but this is fine for now.